test/src/main/java/org/apache/accumulo/test/ZooKeeperPropertiesIT.java

```diff
@@ -38,4 +52,106 @@ public void testNoFiles() {
     }
   }
 
+  @Test
+  @Timeout(30)
+  public void testTablePropUtils() throws AccumuloException, TableExistsException,
+      AccumuloSecurityException, TableNotFoundException {
+    ServerContext context = getServerContext();
+
+    try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
+
+      final String tableName = getUniqueNames(1)[0];
+      client.tableOperations().create(tableName);
+      Map<String,String> idMap = client.tableOperations().tableIdMap();
+      String tid = idMap.get(tableName);
+
+      Map<String,String> properties = client.tableOperations().getConfiguration(tableName);
+      assertEquals("false", properties.get(Property.TABLE_BLOOM_ENABLED.getKey()));
+
+      context.tablePropUtil().setProperties(TableId.of(tid),
+          Map.of(Property.TABLE_BLOOM_ENABLED.getKey(), "true"));
+
+      // add a sleep to give the property change time to propagate
+      properties = client.tableOperations().getConfiguration(tableName);
+      while (!properties.get(Property.TABLE_BLOOM_ENABLED.getKey()).equals("true")) {
+        try {
+          Thread.sleep(250);
+        } catch (InterruptedException e) {
+          fail("Thread interrupted while waiting for tablePropUtil update");
+        }
+        properties = client.tableOperations().getConfiguration(tableName);
+      }
+
+      context.tablePropUtil().removeProperties(TableId.of(tid),
+          List.of(Property.TABLE_BLOOM_ENABLED.getKey()));
+
+      properties = client.tableOperations().getConfiguration(tableName);
+      while (!properties.get(Property.TABLE_BLOOM_ENABLED.getKey()).equals("false")) {
```

Comment:
Could this be simplified to `while (properties.get(Property.TABLE_BLOOM_ENABLED.getKey()).equals("true"))`?